apps/meeting/src/containers/MeetingControls/index.tsx

```diff
@@ -6,7 +6,7 @@ import {
   ControlBar,
   AudioInputVFControl,
   AudioInputControl,
-  VideoInputControl,
+  VideoInputBackgroundBlurControl,
```

Comment:
We do not want to remove the existing `VideoInputControl`, but provide a check to use background blur feature like we do for VF now.

Reply:
done